src/instruments/src/SD/Pages/Cond/Cond.tsx

```diff
@@ -17,71 +17,60 @@
  */
 
 import './Cond.scss';
-import { useState } from 'react';
-import { getSimVar, useUpdate } from '../../../util.js';
+import ReactDOM from 'react-dom';
+import React, { useEffect, useState } from 'react';
+import { renderTarget } from '../../../Common/defaults';
+import { SimVarProvider, useSimVar } from '../../../Common/simVars';
 
 export const CondPage = () => {
-    const [ckptTemp, setCkptTemp] = useState({
-        selectedTemp: 99,
-        trimTemp: 99,
-        cabinTemp: 99,
-        rotation: 0,
-    });
-
-    const [fwdTemp, setFwdTemp] = useState({
-        selectedTemp: 99,
-        trimTemp: 99,
-        cabinTemp: 99,
-        rotation: 0,
-    });
-
-    const [aftTemp, setAftTemp] = useState({
-        selectedTemp: 99,
-        trimTemp: 99,
-        cabinTemp: 99,
-        rotation: 0,
-    });
-
-    const [hotAir, setHotAir] = useState(false);
-
     // Disaply trim valve position for each zone
     const gaugeOffset = -50; // Gauges range is from -50 degree to +50 degree
 
-    useUpdate(() => {
-        setCkptTemp({
-            selectedTemp: getSimVar('L:A320_Neo_AIRCOND_LVL_1', 'number'),
-            trimTemp: getSimVar('L:A32NX_CKPT_TRIM_TEMP', 'celsius').toFixed(0),
-            cabinTemp: getSimVar('L:A32NX_CKPT_TEMP', 'celsius').toFixed(0),
-            rotateTemp: gaugeOffset + ckptTemp.selectedTemp,
-        });
-        setFwdTemp({
-            selectedTemp: getSimVar('L:A320_Neo_AIRCOND_LVL_2', 'number'),
-            trimTemp: getSimVar('L:A32NX_FWD_TRIM_TEMP', 'celsius').toFixed(0),
-            cabinTemp: getSimVar('L:A32NX_FWD_TEMP', 'celsius').toFixed(0),
-            rotateTemp: gaugeOffset + fwdTemp.selectedTemp,
-        });
-        setAftTemp({
-            selectedTemp: getSimVar('L:A320_Neo_AIRCOND_LVL_3', 'number'),
-            trimTemp: getSimVar('L:A32NX_AFT_TRIM_TEMP', 'celsius').toFixed(0),
-            cabinTemp: getSimVar('L:A32NX_AFT_TEMP', 'celsius').toFixed(0),
-            rotateTemp: gaugeOffset + aftTemp.selectedTemp,
-        });
-        setHotAir(getSimVar('L:A32NX_AIRCOND_HOTAIR_TOGGLE', 'bool'));
-    });
+    const [cockpitSelectedTemp] = useSimVar('L:A320_Neo_AIRCOND_LVL_1', 'number', 1000);
+    const [cockpitTrimTemp] = useSimVar('L:A32NX_CKPT_TRIM_TEMP', 'celsius', 1000);
+    const [cockpitCabinTemp] = useSimVar('L:A32NX_CKPT_TEMP', 'celsius', 1000);
+
+    const [cockpitRotateTemp, setCockpitRotateTemp] = useState(0);
+    useEffect(() => {
+        const rotate : number = gaugeOffset + cockpitSelectedTemp;
+        setCockpitRotateTemp(rotate);
+    }, [cockpitSelectedTemp]);
```

Comment:
This is not great, but of course this will be replaced by a real COND system simulation in future.

Reply:
Agree. Open to alternatives if there are any.

Reply:
For now you could just drop the duplicate state and just calculate `gaugeOffset + cockpitSelectedTemp` on every render. In future we probably won't have useEffect here either, but a separate, second LVar.